Juego.py

```diff
@@ -948,7 +948,7 @@ def __init__(self, fontName, size, text, color, lx, ly):
         self.__positionY = ly
 
     def doPaint(self):
-        text = unicode(self.__text, "UTF-8")
+        text = str(self.__text.encode(), "UTF-8")
```

Comment:
What is the purpose of this line?

Reply:
In Python 3, Unicode is replaced with strings because of its abundance, that's why while porting all Unicode calls are replaced with str calls.
`self.__text` here is a string so to convert it into bytes-like object, I used `encode()`.